draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -572,440 +572,443 @@ demonstrating some of the following functionality:
         <t>- flexibility in marking dscp or metadata at ingress and/or egress. </t>
         <figure>
           <artwork><![CDATA[
-   module example-compa-diffserv {
```

Comment:
This is a very painful way of keeping examples up to date. Instead use my macro as follows

INSERT_TEXT_FROM_FILE(../src/yang/example-compa-diffserv.yang)

that takes the name of the file (that you have already created in src/yang/example-compa-diffserv.yang), and it will include the file into the draft. You do not have to cut and paste it every time the file is updated.

Search for the above macro on how to use it.